src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-sentry_sdk.py

```diff
@@ -18,3 +19,9 @@
                  "sentry_sdk.integrations.argv",
                  "sentry_sdk.integrations.logging",
                  "sentry_sdk.integrations.threading"]
+
+if hasattr(si, '_AUTO_ENABLING_INTEGRATIONS'):
+    def make_integration_name(integration_name: str):
+        return ".".join(integration_name.split(".")[:-1])
```

Comment:
Marginally clearer IMO.
```suggestion
        return integration_name.rsplit(".", maxsplit=1)[0]
```